ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -309,9 +309,9 @@ public struct Avatar: View {
                              alignment: .center)
                     .modifyIf(shouldDisplayPresence, { thisView in
                             thisView
-                                .clipShape(PresenceCutout(originX: presenceCutoutOriginX,
-                                                          originY: presenceCutoutOriginY,
-                                                          presenceIconOutlineSize: presenceIconOutlineSize),
+                                .clipShape(CircleCutout(xOrigin: presenceCutoutOriginX,
+                                                          yOrigin: presenceCutoutOriginY,
```

Comment:
nit: is the param alignment correct here? GitHub's interface shows them slightly moved to the right.